.github/workflows/gh-lint.yaml

```
@@ -0,0 +1,152 @@
+      cancel-in-progress: true
+    
+    jobs:
+      lint:
+        runs-on: ubuntu-latest
+        timeout-minutes: 5
+        permissions:
+          contents: read
+        steps:
+          - name: Checkout
+            uses: actions/checkout@eef61447b9ff4aafe5dcd4e0bbf5d482be7e7871 # v4.2.1
+    
```

yamllint

[error] trailing spaces

(trailing-spaces)